src/shared/telemetry/telemetryEvent.ts

```diff
@@ -14,20 +14,28 @@ export interface TelemetryEvent {
 
 export function toMetricData(array: TelemetryEvent[]): MetricDatum[] {
     return ([] as MetricDatum[]).concat(
-        ...array
+        ...(array
```

Comment:
Could we just iterate over this with a for...of loop? We're iterating 3x over the array and we have to coerce the type in two places.